prow/plugins/label/label.go

```diff
@@ -78,7 +78,36 @@ func getLabelsFromREMatches(matches [][]string) (labels []string) {
 	return
 }
 
-func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent) error {
+func getLabelsFromGenericMatches(matches [][]string, allowedLabels []plugins.Label) (labels []string) {
+	if len(allowedLabels) == 0 {
+		return nil
+	}
+out:
```

Comment:
Prefer functions and `return ` over labels?

Reply:
I have simplified the plugin config which obsoletes the need for this.